src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -119,6 +169,7 @@ protected override bool ValidateParameters()
 
                 if (!File.Exists(_inputAssembly))
                 {
+                    Log.LogError(Strings.InputAssemblyNotFound, _inputAssembly);
```

Comment:
Do we want to also check that `_outputR2RImage` is not `null` or an empty string?  Note that in this branch it is a metadata and not an `ItemSpec`.

Reply:
Good point, added in 6th commit.